### PR TITLE
[Easy] Small changes needed to get cargo publish to work

### PR DESCRIPTION
**Pull request by nlordell**
Just some small alterations to some of the crate metadata for publishing on crates.io.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ license = "MIT"
 repository = "https://github.com/gnosis/ethcontract-rs"
 homepage = "https://github.com/gnosis/ethcontract-rs"
 documentation = "https://docs.rs/ethcontract"
+keywords = ["web3", "ethereum", "contract", "async"]
```
Comment (bh2smith) [inline Cargo.toml:10]: perhaps "smart contract" is more precise? Do we want to include "solidity" as well?

**Review by bh2smith — APPROVED** [review]
Looks fantastic! Rock on!

File: Cargo.toml (hunk 1 above)
Comment (nlordell) [reply Cargo.toml:10]: I don't want to include `solidity` since it never interfaces directly with it. Maybe `truffle`?

Comment (nlordell) [reply Cargo.toml:10]: I'll add it as a separate PR later when I do some cleanup (like adding badges and getting crates.io to show a better landing page)

Comment (nlordell) [reply Cargo.toml:10]: Created #35

**Pull request merged**